src/gui/optionsdialog.ui

```diff
@@ -1825,7 +1825,7 @@ Manual: Various torrent properties (e.g. save path) must be assigned manually</s
                    <property name="minimumSize">
```

Comment:
Please try resetting this field to the default.
It works for me on linux.

Reply:
If you mean - reset the default `height` value, then yes I've reset this to `25` & it's like the left window of screenshot in the OP, at least on `windows`

Reference:
https://github.com/xavier2k6/qBittorrent/actions/runs/1927073798#artifacts

It's been like that for ages, one thing I noticed though is if I was to run say official qBittorrent 4.4.1 with command `qbittorrent -style fusion` the height difference is there but isn't as noticeable/doesn't pop out.

Reply:
>If you mean - reset the default height value, then yes I've reset this to 25 & it's like the left window of screenshot in the OP, at least on windows

No. I mean removing this whole `property` block so that it will use the implicit default values from Qt.
Well I also tested on Windows and it looks fine so please remove this whole `property` block.

Reply:
`property` block removed, thank you very much for the tip.